cloudbuild.yaml

```diff
@@ -56,7 +56,11 @@ steps:
   - 'CLOUD_BUILD_BRANCH_NAME=${_CLOUD_BUILD_BRANCH_NAME}'
   - 'DPU_IMAGE_NAME=${_DPU_IMAGE_NAME}'
   - 'CLOUD_BUILD_PAT_TRIGGER_ID=${_CLOUD_BUILD_PAT_TRIGGER_ID}'
+  - 'SECRET_KEY=${_SECRET_KEY}'
```

Comment:
ye kaunsa secret_key hain? main se pull liya hain aapne?

Reply:
remove this line.

Reply:
done